commands.json

```
@@ -4581,6 +4617,38 @@
     "since": "5.0.0",
     "group": "sorted_set"
   },
+  "ZMPOP": {
+    "summary": "Remove and return members with scores in a sorted set",
+    "complexity": "O(K) + O(N*log(M)) where K is the number of provided keys, N being the number of elements in the sorted set, and M being the number of elements popped.",
+    "arguments": [
+      {
+        "name": "numkeys",
+        "type": "integer"
+      },
+      {
+        "name": "key",
+        "type": "key",
+        "optional": true,
+        "multiple": true
+      },
```

Since the number of keys must be gt 0, the first key is actually mandatory.

```suggestion
      {
        "name": "key",
        "type": "key",
      },
      {
        "name": "key",
        "type": "key",
        "optional": true,
        "multiple": true
      },
```

---

Right... we may also need to fix LMPOP/BLMPOP.  The `optional` one and the BLMPOP examples ?

https://github.com/redis/redis-doc/blob/master/commands.json#L441
https://github.com/redis/redis-doc/blob/master/commands/blmpop.md